scene/gui/color_picker.cpp

```diff
@@ -607,10 +616,18 @@ void ColorPicker::set_picker_shape(PickerShapeType p_shape) {
 	if (p_shape == current_shape) {
 		return;
 	}
-	shape_popup->set_item_checked(current_shape, false);
-	shape_popup->set_item_checked(p_shape, true);
+	if (current_shape != SHAPE_NONE) {
+		shape_popup->set_item_checked(current_shape, false);
```

Comment:
Does this affect anything? If the shape button is hidden, the popup can't be opened, so it doesn't really matter what it has selected.

Reply:
I was getting index errors since the popup item didn't include an entry for the none shape.

Reply:
Ah, right..